src/transformStateWithClones.js

```diff
@@ -8,6 +8,33 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const arr = [];
+  let copyOfState = { ...state };
```

Comment:
Also you could use Object assign with all of key values as a second param to create a copy of the state. It's not mistake, just another solve